pkg/hook/accessrequesthook/accessrequesthook.go

```diff
@@ -166,6 +166,19 @@ func (h Hook) NoEntitlementAccess(ctx context.Context, cfg *config.Context, inpu
 			req.Justification.Reason = &customReason
 		}
 	}
+
+	for i := range req.Entitlements {
+
+		if input.Duration != nil {
+			req.Entitlements[i].Duration = input.Duration
+
+		} else if result.DurationConfiguration.DefaultDuration != nil {
```

Comment:
Looks like there needs to be another check here for DurationConfiguration being set to avoid a nil pointer
`if res.Msg.DurationConfiguration != nil`